contract_comm/election/election.go

```diff
@@ -54,3 +124,85 @@ func GetElectedValidators(header *types.Header, state vm.StateDB) ([]common.Addr
 	}
 	return newValSet, nil
 }
+
+type voteTotal struct {
+	Group common.Address
+	Value *big.Int
+}
+
+func getTotalVotesForEligibleValidatorGroups(header *types.Header, state vm.StateDB) ([]voteTotal, error) {
```

Comment:
I think this is the first time we've introduced the notion of a group in celo-blockchain -- previously just limited to the contracts. Not saying it's a problem per se, just a layering observation.